### ISSUE TITLE
Images cannot be shown.

### ISSUE THREAD

**Issue by zbelial** [chenyanming/shrface#10]
Hi, images don't show after enabling this package. Could you confirm and fix(if it really is this package's bug) it? thanks. 

# Emacs version:
 27.0.91
# shrface version:
 20200507 from melpa
# Config: 
```elisp 
(use-package shr
  :ensure nil
  :config (progn
            (setq-default shr-inhibit-images t)   ; toggle with `T`
            (setq-default shr-use-fonts nil)      ; toggle with `F`            
            (setq-default shr-use-colors nil)      ; toggle with `F`            

            (use-package shrface
              :ensure t
              ;; :disabled t
              :config (progn
                        (shrface-basic)
                        ;; (add-hook 'shrface-mode-hook #'(lambda() (org-indent-mode -1)))

                        (with-eval-after-load 'eww
                          (add-hook 'eww-after-render-hook 'shrface-mode)
                          (define-key eww-mode-map (kbd "C-c C-f") 'org-cycle)
                          )
                        )
              )
            )
  )
```
Images wouldn't be loaded by default and I use `T` to toggle the display of images.  Following is the function:
```elisp
            (defun zjy/eww-toggle-images ()
              "Toggle whether images are loaded and reload the current page fro cache."
              (interactive)
              (setq-local shr-inhibit-images (not shr-inhibit-images))
              (eww-reload t)
              (message "Images are now %s"
                       (if shr-inhibit-images "off" "on")))
```

# How to reproduce:
1. enable package shrface
2. eww to visit `https://github.com/chenyanming/shrface`
3. enter `T` to try to load and show images

**Comment by chenyanming** [comment]
@zbelial Thanks your feedback. Check the fixed commit c7deb9f.

**Comment by zbelial** [comment]
Works now, thanks.
